src/ddmd/backend/gflow.c

```diff
@@ -323,7 +323,7 @@ STATIC void accumrd(vec_t GEN,vec_t KILL,elem *n)
                 rdelem(&Gl,&Kl,n->E1);
                 rdelem(&Gr,&Kr,n->E2);
 
-                switch (el_returns(n->E1) * 2 | el_returns(n->E2))
+                switch (el_returns(n->E1) * 2 | int(el_returns(n->E2)))
```

Comment:
This looks strange - why is the cast needed?

Reply:
el_returns returns a boolean which causes:
warning C4805: '|' : unsafe mix of type 'int' and type 'bool' in operation

Reply:
ok